File: Java/Spring/Security-DB-Auth/src/main/java/com/github/spdeepak/security/entity/Role.java
package com.github.spdeepak.security.entity;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Data
@Entity
public class Role {

    @Id
    @GeneratedValue
    private Long id;

    private String role;

}

File: Kotlin-Spring-Security/src/main/kotlin/com/github/spdeepak/entity/Role.kt
package com.github.spdeepak.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

/**
 * @author Deepak Sunanda Prabhakar
 */
@Entity
data class Role(var rolename: String = "") {

	@Id
	@GeneratedValue
	val id: Long? = null

}